docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -490,77 +533,22 @@ myClient.Connection.RpcMetadata = new Dictionary<string, string>()
-
-#### Using the Cloud UI
-
-1. Navigate to https://cloud.temporal.io/settings/identities
-2. Click the Create API Key button
-3. Click Service Account at the top of the page under "Create an API Key for*"
-4. Select the Service Account you would like to create an API Key in the “Mapped to identityˮ input box.
-   Existing Service Accounts are provided in the “Mapped to Identityˮ box.
-5. Add values for the API Key. This includes:
-
-- name (required)
-- description (optional)
-- Expiration date (required - defaulted to 30 days)
-- The current max expiration duration for API Keys is 90 days.
-
-6. Click the Generate API Key button
-7. You will see the new API keyʼs secret, if the API Key was created successfully.
-   Copy and store the secret somewhere safe.
-
-#### Using tcld
-
-To create an API Key for a Service Account, use `tcld apikey create` with the `--service-account-id` flag:
-
-```
-tcld apikey create -n saAK1 --desc "Service Account API Key" --duration 30d --service-account-id "2f68507677904e09b9bcdbf93380bb95"
-```
-
-API Keys for Service Accounts appear in the list of API Keys in the UI and tcld.
-
-### Update an API Key for a Service Account
-
-Global Administrators have ability to disable/enable individual API Keys for any user in their account using the Cloud UI or tcld.
-
-#### Using the Cloud UI
-
-1. Navigate to https://cloud.temporal.io/settings/identities
-2. Find the identity that owns the API Key to disable/enable
-3. Click the Disable/Enable button to perform the desired action.
-   There may be a delay after changing the status of an API Key.
-4. Once successful, the API Keys status will be reflected in the row.
-
-#### Using tcld
+### tcld
 
-Use the `tcld apikey disable` or `tcld apikey enable` command, to disable or enable an API Key.
-
-The process for enabling/disabling an API Key is the same for a User or a Service Account.
+To use an API Key with tcld, use one of the following methods:
 
-### Delete an API Key for a Service Account
+- `--api-key` flag
+- `TEMPORAL_CLOUD_API_KEY` environment variable
 
-Global Administrators have ability to delete individual API Keys for any user in their account using the Cloud UI or tcld.
+### The Cloud Ops API
 
-#### Using the Cloud UI
+To use an API Key with the [Cloud Ops API](/ops), securely pass your API Key in your Cloud Ops API client.
 
-1. Navigate to https://cloud.temporal.io/settings/identities
-2. Find the identity that owns the API Key to disable/enable and click on the row to see the API Keys associated with the identity.
-3. Click the Delete button. There may be a delay after changing the status of an API Key.
-4. Once successful, the API Keys status will be reflected in the row.
+For a complete example, see [Cloud Samples in Go](https://github.com/temporalio/cloud-samples-go/blob/1dd4254b6ed1937e361005c0144410e72b8a5542/client/api/apikey.go).
 
-#### Using tcld
+### The Terraform Provider
```

Comment:
```suggestion 
### Terraform Provider
```

Reply:
Provider is lower-cased throughout our site, e.g.

> ./docs/production-deployment/cloud/terraform-provider.mdx:The GitHub repository for the Terraform provider is [terraform-provider-temporalcloud](https://github.com/temporalio/terraform-provider-temporalcloud/tree/main), where you can find the source code, contribution guidelines, and additional information.